config/config.go

```diff
@@ -25,6 +26,9 @@ func DefaultConfig() *cctypes.Config {
 		AppStore: &cctypes.AppStoreConfig{
 			NumEvmKeysToPrune: 50,
 		},
+		Evm: &cctypes.EvmConfig{
+			GasLimit: math.MaxUint64,
```

Comment:
Should be whatever it is on Ethereum by default

Reply:
We should make it `MaxUint64` so that we don't need to have a feature flag.